packages/tocco-ui/src/EditableValue/typeEditors/DateAbstract.js

```diff
@@ -187,7 +189,7 @@ class DateAbstract extends React.Component {
               icon="times"
               data-clear
               tabIndex={-1}
-              aria-label="clear date"
+              aria-label={this.msg('client-tocco-ui.dateAbstract.dateClear')}
```

Comment:
client-tocco-ui namespace does not exists in imo should not be created. See other tocco-ui components as reference. They use client.component